webcompat/api/endpoints.py

```diff
@@ -128,7 +130,7 @@ def get_issue_category(issue_category):
     # For paginated results on the /issues page,
     # see /issues/search/needstriage.
     elif issue_category == 'new':
-        abort(301)
+        redirect(url_for('.get_issue_category', issue_category='needstriage'), 301)
```

Comment:
@miketaylr 

I have a question here. I am getting response code 404 for 

http://localhost:5000/api/issues/category/new

when I make the changes for redirect.

The redirect I have added in a similar fashion to the /issues/search/new seems to be working fine.

In addition, I also tried adding redirect("https://github.com/", code=301) - to check if redirect to some random URL is working fine. That is also returning me 404 response.

Am I missing something here?